ui/src/app/cohort-search/add-criteria.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input} from '@angular/core';
```

Comment:
Should this component have it's own directory? It's currently the only component without one.